Makefile

```diff
@@ -52,3 +54,8 @@ clean: clean-mockgen
 	go clean ./...
 
 fullbuild: dependencies generate test
+
+install:
```

Comment:
Ah, I tried 'go install binaries/...' and that didn't work. Yeah, your way is better (and it doesn't unconditionally build every time).